daemon/container_operations_unix.go

```diff
@@ -38,7 +33,7 @@ func (daemon *Daemon) setupLinkedContainers(container *container.Container) ([]s
 	var env []string
 	children := daemon.children(container)
 
-	bridgeSettings := container.NetworkSettings.Networks["bridge"]
+	bridgeSettings := container.NetworkSettings.Networks[runconfig.DefaultDaemonNetworkMode().NetworkName()]
```

Comment:
Why not assigning `runconfig.DefaultDaemonNetworkMode().NetworkName()` to a var instead of calling it several time :angel: